.github/workflows/release.yaml

```diff
@@ -0,0 +1,67 @@
+name: release
+on:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
+jobs:
+  build-frontend:
+    name: "build frontend"
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+      - name: install nodejs
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: "ui/.nvmrc"
+      - uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+      - name: install deps
+        run: cd ./ui && npm install
+      - name: build the app
+        run: cd ./ui && npm run build
+      - name: store react production build
+        uses: actions/upload-artifact@v2
+        with:
+          name: react-build
+          path: ui/app/dist
+  release:
+    name: "release"
+    needs: build-frontend
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+      - name: install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17.x
+      - uses: actions/cache@v2
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+      - name: Download react build
+        uses: actions/download-artifact@v2
+        with:
+          name: react-build
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
```

Comment:
I already inject the secrets to be able to push on docker hub.
I'm wondering if we should push the docker image on `quay.io` too